src/sentry/static/sentry/app/index.tsx

```diff
@@ -6,37 +6,36 @@ import {Config} from 'app/types';
 
 const BOOTSTRAP_URL = '/api/client-config/';
 
+function renderApp() {
+  if (document.readyState === 'complete') {
+    window.SentryRenderApp();
+  } else {
+    document.addEventListener('DOMContentLoaded', () => window.SentryRenderApp());
+  }
+}
```

Comment:
@EvanPurkhiser Do you see any issues with this? I'm planning on moving this into a separate PR but with some changes: we don't need to attach `SentryRenderApp` to `window` - I think we can move it into a separate module from `bootstrap` where we can then import + render.

Thoughts?

Reply:
Hmm actually, this may interfere with @scefali's Pipeline rendering...

Reply:
@scefali can we potentially move the pipeline rendering into its own webpack entry? how much of sentry's bootstrapping does it need?

Reply:
@billyvg Another webpack entry for the pipeline makes sense. As far as bootstrapping goes, it only uses `RenderPipelineView`.